apps/base/rucio-daemons/cms-rucio-daemons.yaml

```diff
@@ -76,6 +76,8 @@ conveyorPoller:
   podAnnotations:
     prometheus.io/scrape: "true"
     prometheus.io/port: "8080"
+  database:
+    pool_size: 10
```

Comment:
You need to put this under a config heading (see the Kronos example I sent you yesterday)